### PR TITLE
- added `scripts`

### PR DESCRIPTION
**Pull request by neel**
- added SCRIPTS 
- added EMAIL

```
add_wheel(my_lib
  EMAIL "email@address.com"
  ...
  SCRIPTS
    /path/to/python/script
)
```

### CODE REVIEW

File: python-wheel.cmake [hunk 1]
```diff
@@ -57,6 +58,10 @@ function (add_wheel WHEEL_TARGET)
     set(WHEEL_URL "")
   endif()
 
+  if (NOT WHEEL_URL)
```
Comment (josephbirkner) [inline python-wheel.cmake:61]: I believe this check is supposed to be for WHEEL_EMAIL?

**Review by josephbirkner — CHANGES_REQUESTED** [review]
Thanks for your contribution! Looks great overall, I would be very happy to merge this. Can you take a look at my comment?

File: python-wheel.cmake (hunk 1 above)
Comment (neel) [reply python-wheel.cmake]: Yes, Thanks. I forgot to change that.

**Review by neel — COMMENTED** [review]
- fixed

**Review by josephbirkner — APPROVED** [review]
Thanks 🥳

**Pull request merged**